conda_smithy/configure_feedstock.py

```diff
@@ -1195,9 +1195,26 @@ def render_README(jinja_env, forge_config, forge_dir):
             fh.write(line)
 
 
+def generate_gitignore(forge_dir):
+    default_lines = ["*.pyc", "build_artifacts"]
+    gitignore_path = os.path.join(forge_dir, ".gitignore")
+    ignore_lines = default_lines.copy()
+
+    with open(gitignore_path, "r+") as ignore_file:
```

Comment:
What if this file doesn't exist?

Reply:
Good catch, I was confused with w+, since w+ can create the file if not exist, so what about changing it to mode w+?

Reply:
My bad, it is not w+, either. I will correct it